tests/server/server-executeBulkWrite-004.phpt

```diff
@@ -28,11 +28,11 @@ foreach ($writeConcerns as $wc) {
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
 OK: Got MongoDB\Driver\Exception\RuntimeException
-not master
+not %s
```

Comment:
`not %s` seems pretty generic. Would it make sense to use an error code here, or are we concerned that it might not be standardized across all server versions?

I will note that this test is expecting RuntimeException instead of ServerException. That may be because it predates the introduction of ServerException.

Assuming this only affects `server-executeBulkWrite-004.phpt` and `server-executeBulkWrite-007.phpt`, we can punt this to a separate ticket or address it here if you're confident that the code is reliable for any 3.0+ server.

Reply:
Updated to check for `(primary|master)` to be more resilient.

Reply:
I logged [PHPC-1851](https://jira.mongodb.org/browse/PHPC-1851) to revisit changing this to error code checks.